docs/assets/tables/all_inputs.tsv

```diff
@@ -14,6 +14,9 @@ basespace_fetch	basespace_sample_name	String	The BaseSpace sample name is the sa
 basespace_fetch	sample_name	String	The sample name is the sample identifier used in the Terra.bio data table corresponding to the metadata associated with the sample to be transferred from BaseSpace		Required	BaseSpace_Fetch	general
 cauris_cladetyper	assembly_fasta	File	The assembly file for your sample in FASTA format		Required	Cauris_CladeTyper	general
 cauris_cladetyper	samplename	String	The name of the sample being analyzed		Required	Cauris_CladeTyper	general
+chroquetas_workflow	assembly_fasta	File	Genome fasta		Required	ChroQueTas	general
```

Comment:
would you update these (assembly fasta, samplename) to match other workflows' descriptions?